docs/content/02-tutorials/_index.md

```diff
@@ -13,3 +13,4 @@ pre: "<b>II ⁃ </b>"
 | 💾 [Memory Scheduling](02-tutorials/03-memory-scheduling.html)              | Schedule using the `--mem` slurm flag.                    |
 | 💰 [Cost Tags](02-tutorials/04-cost-tracking.html)                      | Track job costs in AWS Cost Explorer by user and project. |
 | ⇓ [Downloading](02-tutorials/06-downloading.html)                      | Download files at cluster start. |
+| ⇓ [Custom Domain Name](02-tutorials/07-custom-domain-name.html)                      | Setup custom domain name. |
```

Comment:
Needs a different emoji, so I asked chatGPT for some suggestions and it replied with:

> The emoji for a URL could be the "globe" emoji 🌐. This emoji often represents the internet, the world, or global connectivity. Alternatively, you could use the "link" emoji 🔗, which is often used to represent a hyperlink or a connection between two things.